setup.cfg

```diff
@@ -19,5 +19,3 @@ upload-dir = docs/build/html
 addopts =
     --showlocals
     -v
-    --cov=watchdog
-    --cov-report=term-missing
```

Comment:
Last thing: could you revert those changes? :pray:

Reply:
Yeah, sorry. I've removed them locally because I had to keep scrolling to see pytest results because of the coverage noise.